src/cook-web/src/components/auth/EmailLogin.tsx

```
@@ -2,175 +2,64 @@
-export function EmailLogin({
-  onLoginSuccess,
-  onForgotPassword,
-}: EmailLoginProps) {
-  const { toast } = useToast();
-  const [isLoading, setIsLoading] = useState(false);
-  const [email, setEmail] = useState('');
-  const [password, setPassword] = useState('');
-  const [termsAccepted, setTermsAccepted] = useState(false);
-  const [emailError, setEmailError] = useState('');
-  const [passwordError, setPasswordError] = useState('');
+export function EmailLogin({}: EmailLoginProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)